docs/reference/api.md

```diff
@@ -14,35 +14,35 @@ For more examples, see "How to use Pebble API". <!-- [David] Link to the how-to
 
 To access the API endpoints over the Unix socket, use the `--unix-socket` option of `curl`. For example:
 
-* TODO
-
-    ```
-    curl ...
-    ```
+```{terminal}
+   :input: curl --unix-socket /path/to/.pebble.socket -XPOST http://localhost/v1/services -d '{"action": "stop", "services": ["svc1"]}'
```

Comment:
I think we should consistency use `--long` arguments here (`--request` instead of `-X` and `--data` instead of `-d`). Also, should we drop `-X/--request` given that POST is implied with `-d/--data`?